Header header
geometry_msgs/Pose robot_image_pose
geometry_msgs/Pose[] fly_image_poses
